File: benchmarks/suite/dict_ops.js
const N = 100_000;
const d = new Map();
for (let i = 0; i < N; i++) d.set(`k${i}`, i);
let sum = 0n;
for (let j = 0; j < N; j++) sum += BigInt(d.get(`k${j}`));
console.log(sum.toString());

File: benchmarks/suite/dict_ops.go
package main

import "fmt"

func main() {
	const N = 100_000
	d := make(map[string]int, N)
	for i := 0; i < N; i++ {
		d[fmt.Sprintf("k%d", i)] = i
	}
	var sum int64
	for j := 0; j < N; j++ {
		sum += int64(d[fmt.Sprintf("k%d", j)])
	}
	fmt.Println(sum)
}
